diff -ur u-boot-2021.04-orig/drivers/usb/host/xhci-brcm.c u-boot-2021.04/drivers/usb/host/xhci-brcm.c
--- u-boot-2021.04-orig/drivers/usb/host/xhci-brcm.c	2021-05-17 01:19:30.199468824 -0300
+++ u-boot-2021.04/drivers/usb/host/xhci-brcm.c	2021-05-17 01:20:28.608833631 -0300
@@ -82,6 +82,7 @@
 
 static const struct udevice_id xhci_brcm_ids[] = {
 	{ .compatible = "brcm,generic-xhci" },
+	{ .compatible = "generic-xhci" },
 	{ }
 };
 
